deploy/cdk/lib/static-host/deployment-pipeline.ts

```diff
@@ -0,0 +1,261 @@
+import { BuildSpec, LinuxBuildImage, PipelineProject } from '@aws-cdk/aws-codebuild'
+import codepipeline = require('@aws-cdk/aws-codepipeline')
```

Comment:
Strictly a style comment, so I won't request changes...

In L2-L5 here, you're importing the entire module and also individual classes from the modules. My preference would be to only do one of these (with a stronger preference for only needed classes).

Reply:
I can see that. I think in general I have focused more on code readability versus import consistency. For instance, some packages have ambiguous class names like `StringParameter`, which you wouldn't necessarily know is referring to parameter store in ssm unless you import the whole module and thus write out your code `ssm.StringParameter`. (Other examples include `codebuild.Project, lambda.Code, dynamodb.Table,` etc.) You could alias your imports, but I think that would make it more confusing if you are already familiar with the CDK construct names and you see something unfamiliar because it has been aliased.

...But there is also plenty of times where I've copied from another file and paid no notice to which ones they imported which way, which may be what happened here. It's probably worth paying a little more attention to.